### PR TITLE
Dev/clone task

### PR DESCRIPTION
**Pull request by RohitVarma001**
### Functionality:
Clonning a Task

### Solution:
Copy a Task which is already created

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
firebase emulators : start
ng serve

### CODE REVIEW

File: src/app/body/task-details/clone-task/clone-task.component.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    const callable = this.functions.httpsCallable('createNewTask');
+
+    try {
+      const result = await callable({ Title: this.title, Description: this.description, Priority: this.priority, Difficulty: this.difficulty, Creator: this.creatorName, Assignee: this.assigneeName, EstimatedTime: this.estimatedTime, Status: this.status, Category: this.category, SprintNumber: this.sprintNumber, StoryPointNumber: this.storyPoint, CreationDate: this.todayDate, Time: this.time }).toPromise();
+
+      console.log("Successfully created the task");
+      console.log(result);
+      //this.router.navigate(['/']);
+      this.showClose = true;
+      //this.cloneTaskDone();
```
Comment (shivamgoyal7) [inline src/app/body/task-details/clone-task/clone-task.component.ts:95]: remove the commented line

File: src/app/body/task-details/clone-task/clone-task.component.ts [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  storyPoint: number
+  time: string
+  enableLoader: boolean = false
+  valid: boolean = true;
+
+  constructor(private functions: AngularFireFunctions, public validationService: ValidationService, private router: Router, private location: Location, public toolsService: ToolsService, public navbarHandler: NavbarHandlerService, public errorHandlerService: ErrorHandlerService) { }
+
+  ngOnInit(): void {
+    // this.navbarHandler.resetNavbar();
+    // this.navbarHandler.addToNavbar(this.componentName);
```
Comment (shivamgoyal7) [inline src/app/body/task-details/clone-task/clone-task.component.ts:46]: remove commented line

File: src/app/body/task-details/task-details.component.ts [hunk 3]
```diff
@@ -72,7 +76,12 @@ export class TaskDetailsComponent implements OnInit {
     this.editTaskEnabled = false;
   }
 
+  cloneTaskCompleted(data: {completed: boolean}){
+    this.cloneTaskEnabled = false;
+  }
+
   async deleteTask() {
+
```
Comment (shivamgoyal7) [inline src/app/body/task-details/task-details.component.ts:84]: remove this extra space

**Review by shivamgoyal7 — COMMENTED** [review]
remove commented line

**Comment by RohitVarma001** [comment]
Now @shivamgoyal7 Can You Check

**Comment by viv9k** [comment]
I wanted demo on this tomorrow @RohitVarma001 Please schedule the meeting for tomorrow evening at 7:00

**Comment by RohitVarma001** [comment]
Ok

**Comment by viv9k** [comment]
Demo is done and changes are requested.

**Review by viv9k — APPROVED** [review]
Ship it

**Pull request merged**